filebeat/module/elasticsearch/slowlog/test/es74_index_search_slowlog-json.log-expected.json

```diff
@@ -19,7 +19,7 @@
         "event.dataset": "elasticsearch.slowlog",
         "event.duration": 6000000,
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
Similar to my other comment: https://github.com/elastic/beats/pull/13874/files#r330526232.